tests/e2e/Identity.test.ts

```diff
@@ -65,6 +71,69 @@ describe('e2e', () => {
       })
     })
 
+    describe('Update identity', () => {
+      let newIdentityWallet
+
+      before(async () => {
+        // make sure we have a wallet with a proper implementation and deployed identity
+        const walletData = await identityWallet.create()
+        await identityWallet.loadFrom(walletData)
+        await identityWallet.deployIdentity()
+
+        // create a new identity with a different implementation address
+        newIdentityWallet = new IdentityWallet(
+          relayProvider,
+          tlNetworkConfigIdentity.chainId,
+          tlNetworkConfigIdentity.identityFactoryAddress,
+          secondIdentityImplementationAddress,
+          NonceMechanism.Random
+        )
+        await newIdentityWallet.loadFrom(walletData)
+
+        expect(
+          await newIdentityWallet.getIdentityImplementationAddress()
+        ).to.equal(identityImplementationAddress)
+        expect(
+          await newIdentityWallet.isIdentityImplementationUpToDate()
+        ).to.equal(false)
```

Comment:
You assert that the setup of your test is correct, we did not test anything yet.

This is useful because if the setup is wrong, the output of the tests will tell you that the `before` hook failed, whereas if you put the expect in the test, you will see for example that `should confirm an update of identity implementation` failed while the failure has nothing to do with confirming an update.

Reply:
Hm, okay I see your point. 
It would make it a little more specific if we had a separate test for this, but on the downside - if the test don't run in order, but a run in parallel - you'll run into problems. So yes, your assert looks strange there, but makes sense.